build/.vsts-ci.yml

```diff
@@ -5,7 +5,7 @@ resources:
 jobs:
 - job: checksubmodule
   pool:
-    name: Hosted Ubuntu 1604
+    vmImage: 'ubuntu-20.04'
```

Comment:
this is moving on from self hosted right?

Reply:
Yes, correct, we are using Azure DevOps agents for that.